azure-pipelines.yml

```diff
@@ -57,7 +57,7 @@ jobs:
           projectDirectory: '$(Build.SourcesDirectory)\react-components'
           arguments: test --ci --watchAll=false --reporters=jest-junit --reporters=default --coverage --coverageReporters=cobertura --testMatch "**\*.spec.[t]s?(x)"
         displayName: Run tests
-        continueOnError: true # Test failures should be published before failing the build
+        continueOnError: false # Test failures should fail the build
```

Comment:
As you can see from the comment, the intent of this line was that the next task should run, and publish the details of any error to the Tests pane of the pipeline. I'd suggest adding `condition: always()` to the next task so that it runs even if this fails, and then test that that works.

Reply:
Issue addressed in new PR